loleaflet/src/control/Control.MobileWizard.js

```diff
@@ -516,8 +516,10 @@ L.Control.MobileWizard = L.Control.extend({
 				this._scrollToPosition(lastScrollPosition);
 			}
 
-			if (isPopup)
+			if (isPopup) {
+				$('#mobile-wizard').addClass('jsdialog');
```

Comment:
this line will add class forever for #mobile-wizard, won't it break anything?
maybe add `$('#mobile-wizard').removeClass('jsdialog');` to `reset` function?
or don't add jsdialog at all, see other comment

Reply:
yup, you are right I will add that remove Class and change the additional class